src/implementation/data-elements-and-custom-dimensions.md

```diff
@@ -122,14 +146,81 @@ renders these combinations invalid and a low-level database operation
 must be done to correct it. It is hence recommended to thoughtfully
 consider which breakdowns are required and to not change them too often.
 
+## Atrtibute combinations
```

Comment:
typo
```suggestion
## Attribute combinations
```

The way this section is written is clear. My only concern is that in the system we don't really have things called "attribute combinations". We have category combinations where the categories have the data dimension type of "Attribute". 
That is addressed partly [here](https://docs.dhis2.org/en/use/user-guides/dhis-core-version-master/configuring-the-system/metadata.html#use_category_combo_for_data_set). 
I wonder if we should make sure these are aligned? (Though I completely understand if it is more common, and simpler, for implementers to talk about "attribute combinations").